deploy/cloudformation/elastic-agent-ec2-cnvm.yml

```diff
@@ -3,8 +3,8 @@ AWSTemplateFormatVersion: "2010-09-09"
 Description: Creates an EC2 instance with permissions to run elastic-agent
 Parameters:
   LatestAmiId:
-    Type: 'AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>'
-    Default: '/aws/service/ami-amazon-linux-latest/al2023-ami-minimal-kernel-default-arm64'
+    Type: "AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>"
```

Comment:
I feel like there's an IDE formatting war between some of the CF commuters (see [here](https://github.com/elastic/cloudbeat/commit/74421216ee7a0dbee4746e9571b55567a0ea49f3#) for example lol) Please let's solve this instead of committing multiple `"`->`'` lines

Reply:
[`rain fmt`](https://github.com/aws-cloudformation/rain) seems to be the AWS recommendation.